neurom/io/utils.py

```diff
@@ -116,7 +116,7 @@ def load_neuron(filename, tree_action=None):
         raise NonConsecutiveIDsError('Non consecutive IDs found in raw data')
 
     nrn = make_neuron(data, tree_action)
-    nrn.id = os.path.splitext(filename)[0]
+    nrn.id = os.path.splitext(filename)[0].split('/')[-1]
```

Comment:
Can I use both os.path.splitext and os.path.basename in one operation or should I use them consecutively?